data_collection/gazette/spiders/rj/rj_japeri.py

```diff
@@ -0,0 +1,28 @@
+from datetime import date
+
+import dateparser
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RjJaperiSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3302270"
+    name = "rj_japeri"
+    allowed_domains = ["pmjaperi.geosiap.net.br"]
+    start_date = date(2019, 7, 22)
```

Comment:
Opa essa data seria a primeira de acordo com a coluna de data de publicação da tabela, mas parece que esta coluna não pode ser confiada, nem o atributo `dt_publicacao` do json :-|

A data contida no nome do arquivo parece refletir a data de publicação correta.
Sendo assim, a data inicial seria 03/06/2019
```suggestion
    start_date = date(2019, 6, 3)
```